### PR TITLE
Update Chinese (Simplified) translations for version 2.1.0rc1

### PR DESCRIPTION
**Pull request by ghost**
This PR will stay as a work in progress until version 2.1.0rc1 is ready. In other words - please merge this PR right before merging #5834.

### Checklist (from #5834)
- [x] Moderator role: #5643 
- [x] Audit log: this PR
- [x] Lists API: N/A
- [x] Account migration: #5764, #5816 and this PR
- [x] Invite system: this PR
- [x] Lists UI: (moved to #5891)
- [x] Optionally hide reblogs from followed users: this PR

Last minute amendments:
- [x] Fix extra space between status text & username: N/A
- [x] `Fix` #5856: N/A
- [x] Add clean-up for redis parts of lists when lists are deleted (lists of inactive users?): N/A

----------

@Artoria2e5 please review

### CODE REVIEW

**Comment by ykzts** [comment]
cc @Artoria2e5 @KenOokamiHoro @leopku @miuchan @vidarlee

File: app/javascript/mastodon/locales/zh-CN.json [hunk 1]
```diff
@@ -102,23 +104,23 @@
...
+  "keyboard_shortcuts.column": "使某一栏中的嘟文获得焦点",
+  "keyboard_shortcuts.compose": "使嘟文撰写框获得焦点",
+  "keyboard_shortcuts.description": "说明",
+  "keyboard_shortcuts.down": "在列表中让光标下移",
+  "keyboard_shortcuts.enter": "展开嘟文",
+  "keyboard_shortcuts.favourite": "收藏嘟文",
+  "keyboard_shortcuts.heading": "键盘快捷键",
+  "keyboard_shortcuts.hotkey": "快捷键",
+  "keyboard_shortcuts.legend": "显示此说明",
+  "keyboard_shortcuts.mention": "提及嘟文发送者",
```
Comment (Artoria2e5) [inline app/javascript/mastodon/locales/zh-CN.json]: Try 作者/author instead of 发送者/sender

File: config/locales/zh-CN.yml [hunk 2]
```diff
@@ -133,6 +133,32 @@ zh-CN:
...
+        create_email_domain_block: "%{name} 屏蔽了电子邮件域名 %{target}"
+        demote_user: "%{name} 对用户 %{target} 进行了降任操作"
+        destroy_domain_block: "%{name} 解除了对域名 %{target} 的屏蔽"
+        destroy_email_domain_block: "%{name} 解除了对电子邮件域名 %{target} 的屏蔽"
+        destroy_status: "%{name} 删除了 %{target} 的嘟文"
+        disable_2fa_user: "%{name} 停用了用户 %{target} 的双重认证"
+        disable_custom_emoji: "%{name} 停用了自定义表情 %{target}"
+        disable_user: "%{name} 将用户 %{target} 设置为禁止登录"
+        enable_custom_emoji: "%{name} 启用了自定义表情 %{target}"
+        enable_user: "%{name} 将用户 %{target} 设置为允许登录"
```
Comment (Artoria2e5) [inline config/locales/zh-CN.yml]: Shoud we make it more… flexible as in "禁止/允许 %{target} 登录"? And the 用户/user part is probably implied by target's string format

**Review by Artoria2e5 — CHANGES_REQUESTED** [review]
✏️ 🔜 👍

File: config/locales/zh-CN.yml (hunk 2 above)
Comment (SerCom-KC) [reply config/locales/zh-CN.yml]: Yes, this is audit log, see #5757

Comment (SerCom-KC) [reply config/locales/zh-CN.yml]: Plus, IMO `admin 禁止了 user 登录` sounds like a one-time-only thing to me, so despite of being stupid the current translation seems more accurate for a long-term ban

File: app/javascript/mastodon/locales/zh-CN.json (hunk 1 above)
Comment (SerCom-KC) [reply app/javascript/mastodon/locales/zh-CN.json]: Done

**Review by Artoria2e5 — APPROVED** [review]

**Review by ghost — COMMENTED** [review]
Screenshot in English for 34920e395bd79b3722447b626325dccf21bdfbaa

I guess we can't do double-approve on GitHub so comment to approve this commit instead

File: app/javascript/mastodon/locales/zh-CN.json [hunk 3]
```diff
@@ -134,6 +134,7 @@
   "navigation_bar.favourites": "收藏的内容",
   "navigation_bar.follow_requests": "关注请求",
   "navigation_bar.info": "关于本站",
+  "navigation_bar.keyboard_shortcuts": "键盘快捷键",
```
Comment (Artoria2e5) [inline app/javascript/mastodon/locales/zh-CN.json]: "keyboard accelerator key" is redundant. just say 快捷键.

**Review by Artoria2e5 — CHANGES_REQUESTED** [review]
🖊

File: app/javascript/mastodon/locales/zh-CN.json (hunk 3 above)
Comment (SerCom-KC) [reply app/javascript/mastodon/locales/zh-CN.json]: Also apply to `keyboard_shortcuts.heading`? It's the title of the column.

**Comment by SerCom-KC** [comment]
Screenshot in English for `navigation_bar.keyboard_shortcuts` (4e7de9e553a06197e80836b632560439eee06ebb)

File: app/javascript/mastodon/locales/zh-CN.json (hunk 3 above)
Comment (Artoria2e5) [reply app/javascript/mastodon/locales/zh-CN.json]: I think so.

**Review by leopku — APPROVED** [review]

File: app/javascript/mastodon/locales/zh-CN.json [hunk 4]
```diff
@@ -106,20 +106,20 @@
   "home.settings": "栏目设置",
   "keyboard_shortcuts.back": "返回上一页",
   "keyboard_shortcuts.boost": "转嘟",
-  "keyboard_shortcuts.column": "使某一栏中的嘟文获得焦点",
-  "keyboard_shortcuts.compose": "使嘟文撰写框获得焦点",
+  "keyboard_shortcuts.column": "选择第 X 栏中的嘟文",
```
Comment (SerCom-KC) [inline app/javascript/mastodon/locales/zh-CN.json]: I'll explain this one: The key for this is `1` to `9`, so if your first column is "Home", your second column is "Notifications", and your third column is "Federated timeline", press `1` will focus on the first toot in the "Home" column, press `2` will focus on the first toot in the "Notifications" column, and press `3` will focus on the first toot in the "Federated timeline" column. I think `选择第 X 栏中的嘟文` (Select toot in the Xth column) is fairly easy for users to understand.

Comment (Artoria2e5) [reply app/javascript/mastodon/locales/zh-CN.json]: :+1: for column. Focus sounds good for now too.

**Comment by SerCom-KC** [comment]
Translations for Lists UI is now in a separate PR (#5891).
@ykzts I think it's time to remove the WIP tag
@Gargron Please also merge tootsuite/joinmastodon#62? I'd also like to see the styling tweaks in tootsuite/joinmastodon#57 get merged as well.
@Artoria2e5 Anything else? If not please approve. I guess there'll be no further commits in this PR.

**Comment by ykzts** [comment]
Okay. Merge after review by @Artoria2e5.

**Review by Artoria2e5 — APPROVED** [review]

**Review by ykzts — APPROVED** [review]

File: config/locales/zh-CN.yml [hunk 5]
```diff
@@ -394,12 +432,37 @@ zh-CN:
...
+      '1800': 30 分钟后
+      '21600': 6 小时后
+      '3600': 1 小时后
+      '43200': 12 小时后
+      '86400': 1 天后
+    expires_in_prompt: 永不过期
+    generate: 生成邀请链接
+    max_uses: "%{count} 次"
+    max_uses_prompt: 无限制
+    prompt: 生成可供分享的链接以便邀请他人注册本实例
```
Comment (SerCom-KC) [inline config/locales/zh-CN.yml]: Hmm, `以便邀请他人在本实例注册` might be a better one...

Comment (SerCom-KC) [reply config/locales/zh-CN.yml]: @leopku I don't think simplification is required here.

Comment (Artoria2e5) [reply config/locales/zh-CN.yml]: Nah… 以便 in translations usually hints that you are sticking to the original structure too much. Try 生成邀请链接，分享给他人来本实例注册. (Yeah it sucks I know I am exhausted from four hours of sleep per day and this paper shit)

**Pull request merged**